low-level/framework/utils/mon_utils.py

```diff
@@ -28,3 +30,8 @@ def get_alert_id(epoch_time):
     salt = str(uuid.uuid4().hex)
     alert_id = epoch_time + salt
     return alert_id
+
+def get_epoch_time_from_date_and_time(date, time):
```

Comment:
keep name short get_epoch() is enough. Based on arguments, people can infer further, or later we can add/alter arguments too
Also this is generic utility function, can stay in uility.py

Reply:
Done the changes.
Moved the get_epoch() function to Utility.py
Also, Can't we move get_alert_id() func to Utiliy.py too?
why to keep separate mon_utils.py file for that?

Reply:
Monitoring specific ulity functions maintained in mon_utils